Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -99,6 +100,14 @@ async function run() {
         if(webAppUri) {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
+		
+		if(renameFilesFlag){
```

Comment:
Should we even have this check? How will it work if user is disabling an already enabled flag? Don't you have to update the website accordingly?